eclair-core/src/main/scala/fr/acinq/eclair/package.scala

```diff
@@ -120,7 +114,7 @@ package object eclair {
     override def toFloat(x: MilliSatoshi): Float = x.toLong
     override def toDouble(x: MilliSatoshi): Double = x.toLong
     override def compare(x: MilliSatoshi, y: MilliSatoshi): Int = x.compare(y)
-    override def parseString(str: String): Option[MilliSatoshi] = ???
+    override def parseString(str: String): Option[MilliSatoshi] = None
```

Comment:
Wasn't the `???` (which throws an `NotImplementedException`) actually better here?

Reply:
I did that to align with bitcoin-lib, which implements this as `None`, but I can revert.
It's true that since we don't want to use it, we probably want to raise an exception.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1637/commits/9ebf61f61aba31ef08b4f295877f4f7b77c25272